infrastructure/__main__.py

```diff
@@ -217,10 +217,6 @@ def main():
     create_pg_database(resource_group, sql_server)
     app_service_plan = create_app_service_plan(resource_group)
     app_insights = create_app_insights(resource_group)
-    storage_account = create_storage_account(resource_group)
```

Comment:
Likewise with this, doesn't look like a linting removal. Is this related to #104

Reply:
That's an excellent catch. Only meant to remove the assignment into an unused variable (because linting), not the function call. Fixed now.